ugvc/mrd/balanced_strand_utils.py

```diff
@@ -1188,6 +1189,73 @@ def plot_trimmer_histogram(
     return axs
 
 
+def flatten_strand_ratio_category(h5_file: str, key: str) -> pd.DataFrame:
```

Comment:
I am having a hard time following, could you add a docstring and explain briefly why and empty name is required?

Reply:
In the Papyrus json format, the index is an empty string.